### ISSUE TITLE
Failed export base64

### ISSUE THREAD

**Issue by rekurt** [avocode/avocode#1206]
IMPORTANT: Please try to describe how did it happen.

-----
App v. `2.11.1`
Avocode UID: 108517
```
Error: Unknown layer type "layer"
    at e.value (/work/soft/Avocode-linux-x64/resources/app.asar/node_modules/@avocode/rendering/dist/rendering.js:1:14454)
    at e.value (/work/soft/Avocode-linux-x64/resources/app.asar/node_modules/@avocode/rendering/dist/rendering.js:1:8334)
    at e.value (/work/soft/Avocode-linux-x64/resources/app.asar/node_modules/@avocode/rendering/dist/rendering.js:1:7954)
    at e.renderLayer (file:///work/soft/Avocode-linux-x64/resources/app.asar/dist/avocode.min.js:27:12809)
    at file:///work/soft/Avocode-linux-x64/resources/app.asar/dist/avocode.min.js:27:12321
    at /work/soft/Avocode-linux-x64/resources/app.asar/node_modules/immutable/dist/immutable.js:3018:46
    at List.__iterate (/work/soft/Avocode-linux-x64/resources/app.asar/node_modules/immutable/dist/immutable.js:2208:13)
    at IndexedIterable.mappedSequence.__iterateUncached (/work/soft/Avocode-linux-x64/resources/app.asar/node_modules/immutable/dist/immutable.js:3017:23)
    at seqIterate (/work/soft/Avocode-linux-x64/resources/app.asar/node_modules/immutable/dist/immutable.js:606:16)
    at IndexedIterable.IndexedSeq.__iterate (/work/soft/Avocode-linux-x64/resources/app.asar/node_modules/immutable/dist/immutable.js:322:14)
    at IndexedIterable.toArray (/work/soft/Avocode-linux-x64/resources/app.asar/node_modules/immutable/dist/immutable.js:4260:23)
    at new List (/work/soft/Avocode-linux-x64/resources/app.asar/node_modules/immutable/dist/immutable.js:2067:62)
    at reify (/work/soft/Avocode-linux-x64/resources/app.asar/node_modules/immutable/dist/immutable.js:3572:37)
    at List.map (/work/soft/Avocode-linux-x64/resources/app.asar/node_modules/immutable/dist/immutable.js:4403:14)
    at e.renderLayers (file:///work/soft/Avocode-linux-x64/resources/app.asar/dist/avocode.min.js:27:12184)
    at e.copySelectedLayersAsBase64 (file:///work/soft/Avocode-linux-x64/resources/app.asar/dist/avocode.min.js:17:15904)
    at t._handleCopyBase64 (file:///work/soft/Avocode-linux-x64/resources/app.asar/dist/avocode.min.js:24:2428)
    at file:///work/soft/Avocode-linux-x64/resources/app.asar/dist/avocode.min.js:24:688
    at CallbacksRegistry.apply (/work/soft/Avocode-linux-x64/resources/electron.asar/common/api/callbacks-registry.js:54:39)
    at EventEmitter.<anonymous> (/work/soft/Avocode-linux-x64/resources/electron.asar/renderer/api/remote.js:282:21)
    at emitThree (events.js:116:13)
    at EventEmitter.emit (events.js:194:7)
```

**Comment by vojtatranta** [comment]
Hi @rekurt ,
this is a bug for sure.

We are working on fix, it will land in `2.12.0`. Please can you than confirm that this bug was fixed?

**Comment by connor-bean** [comment]
Hey!

We have released the newest version of the Avocode app, version 2.12, which we expect to fix this problem. Please download it at https://avocode.com/download/. I am going to close this ticket but please comment on it if you’re still experiencing this problem in Version 2.12.

Thanks!
Connor, Avocode Support
